Tests/ContentfulTests/DVRRecordings/SyncWithPaginationTests.json

```diff
@@ -0,0 +1,123 @@
ljYXQiLA0KICAgICAgICAidHlwZSI6ICJBc3NldCIsDQogICAgICAgICJjcmVhdGVkQXQiOiAiMjAxOC0wMi0yOFQxNDowNTowNi42NjFaIiwNCiAgICAgICAgInVwZGF0ZWRBdCI6ICIyMDE4LTAyLTI4VDE0OjA1OjA2LjY2MVoiLA0KICAgICAgICAiZW52aXJvbm1lbnQiOiB7DQogICAgICAgICAgInN5cyI6IHsNCiAgICAgICAgICAgICJpZCI6ICJtYXN0ZXIiLA0KICAgICAgICAgICAgInR5cGUiOiAiTGluayIsDQogICAgICAgICAgICAibGlua1R5cGUiOiAiRW52aXJvbm1lbnQiDQogICAgICAgICAgfQ0KICAgICAgICB9LA0KICAgICAgICAicmV2aXNpb24iOiAxDQogICAgICB9LA0KICAgICAgImZpZWxkcyI6IHsNCiAgICAgICAgInRpdGxlIjogew0KICAgICAgICAgICJlbi1VUyI6ICJoYXBweWNhdCINCiAgICAgICAgfSwNCiAgICAgICAgImZpbGUiOiB7DQogICAgICAgICAgImVuLVVTIjogew0KICAgICAgICAgICAgInVybCI6ICIvL2ltYWdlcy5jdGZhc3NldHMubmV0L2R1bXJpM2Via25vbi9oYXBweWNhdC8xY2Q4YzkzNGM5Y2Q5ZTBjZWQ4MTcyOTg0Mzk3M2Y4ZC9oYXBweWNhdHcuanBnIiwNCiAgICAgICAgICAgICJkZXRhaWxzIjogew0KICAgICAgICAgICAgICAic2l6ZSI6IDU5OTM5LA0KICAgICAgICAgICAgICAiaW1hZ2UiOiB7DQogICAgICAgICAgICAgICAgIndpZHRoIjogMjczLA0KICAgICAgICAgICAgICAgICJoZWlnaHQiOiAzOTcNCiAgICAgICAgICAgICAgfQ0KICAgICAgICAgICAgfSwNCiAgICAgICAgICAgICJmaWxlTmFtZSI6ICJoYXBweWNhdHcuanBnIiwNCiAgICAgICAgICAgICJjb250ZW50VHlwZSI6ICJpbWFnZS9qcGVnIg0KICAgICAgICAgIH0NCiAgICAgICAgfQ0KICAgICAgfQ0KICAgIH0sDQogICAgew0KICAgICAgInN5cyI6IHsNCiAgICAgICAgInNwYWNlIjogew0KICAgICAgICAgICJzeXMiOiB7DQogICAgICAgICAgICAidHlwZSI6ICJMaW5rIiwNCiAgICAgICAgICAgICJsaW5rVHlwZSI6ICJTcGFjZSIsDQogICAgICAgICAgICAiaWQiOiAiZHVtcmkzZWJrbm9uIg0KICAgICAgICAgIH0NCiAgICAgICAgfSwNCiAgICAgICAgImlkIjogIm55YW5jYXQiLA0KICAgICAgICAidHlwZSI6ICJBc3NldCIsDQogICAgICAgICJjcmVhdGVkQXQiOiAiMjAxOC0wMi0yOFQxNDowNTowNS41MDJaIiwNCiAgICAgICAgInVwZGF0ZWRBdCI6ICIyMDE4LTAyLTI4VDE0OjA1OjA1LjUwMloiLA0KICAgICAgICAiZW52aXJvbm1lbnQiOiB7DQogICAgICAgICAgInN5cyI6IHsNCiAgICAgICAgICAgICJpZCI6ICJtYXN0ZXIiLA0KICAgICAgICAgICAgInR5cGUiOiAiTGluayIsDQogICAgICAgICAgICAibGlua1R5cGUiOiAiRW52aXJvbm1lbnQiDQogICAgICAgICAgfQ0KICAgICAgICB9LA0KICAgICAgICAicmV2aXNpb24iOiAxDQogICAgICB9LA0KICAgICAgImZpZWxkcyI6IHsNCiAgICAgICAgInRpdGxlIjogew0KICAgICAgICAgICJlbi1VUyI6ICJOeWFuIENhdCINCiAgICAgICAgfSwNCiAgICAgICAgImZpbGUiOiB7DQogICAgICAgICAgImVuLVVTIjogew0KICAgICAgICAgICAgInVybCI6ICIvL2ltYWdlcy5jdGZhc3NldHMubmV0L2R1bXJpM2Via25vbi9ueWFuY2F0L2M3OGFhOTdiZjU1YjdkZTIyOWVlNWE1Zjg4MjYxYWE0L055YW5fY2F0XzI1MHB4X2ZyYW1lLnBuZyIsDQogICAgICAgICAgICAiZGV0YWlscyI6IHsNCiAgICAgICAgICAgICAgInNpemUiOiAxMjI3MywNCiAgICAgICAgICAgICAgImltYWdlIjogew0KICAgICAgICAgICAgICAgICJ3aWR0aCI6IDI1MCwNCiAgICAgICAgICAgICAgICAiaGVpZ2h0IjogMjUwDQogICAgICAgICAgICAgIH0NCiAgICAgICAgICAgIH0sDQogICAgICAgICAgICAiZmlsZU5hbWUiOiAiTnlhbl9jYXRfMjUwcHhfZnJhbWUucG5nIiwNCiAgICAgICAgICAgICJjb250ZW50VHlwZSI6ICJpbWFnZS9wbmciDQogICAgICAgICAgfQ0KICAgICAgICB9DQogICAgICB9DQogICAgfQ0KICBdLA0KICAibmV4dFBhZ2VVcmwiOiAiaHR0cHM6Ly9jZG4uY29udGVudGZ1bC5jb20vc3BhY2VzL2R1bXJpM2Via25vbi9lbnZpcm9ubWVudHMvbWFzdGVyL3N5bmM/c3luY190b2tlbj13NVpHdzZKRndxWm1WY0tzRThLb3c0Z3J3NDVRZHliRHFjS3l3NTA3dzRYRGpzT3Z3cmZEbjhPRVBWME5XOEtjdzRWZ0NFbG9CVF9EdjJYQ214RXFWTU9ud3FKUHdwekRyOEtSdzVwekhCczJ3ckhDaG1BMERoX0N0c09RdzZ2RHU4T193NzNEbno0UHc0SXd3cERDdGNLRHdxcHN3b3JDdWNLNSINCn0NCg==",
```

Comment:
For easyness of verification, do you mind having the content here as text and not as base64? I can easier understand what is happening if it is in clear text.

Reply:
I totally agree. I just followed the current json pattern.

Reply:
I tried to put a JSON in the `body` section but it complains that the response has an invalid format.

Reply:
Okay, so since that seems to be the default of this repository, then let's keep it base64 .... Thanks for trying.